### PR TITLE
Add VERSION + change build string

### PR DESCRIPTION
**Pull request by invadersmustdie**

### CODE REVIEW

**Comment by arnisoph** [comment]
what do you think about using https://github.com/exaring/release-cli ?

**Comment by invadersmustdie** [comment]
release-cli is nice, but a bit too much for this project at this point.

File: Makefile [hunk 1]
```diff
@@ -5,7 +5,7 @@ TARGET := $(shell echo $${PWD\#\#*/})
 .DEFAULT_GOAL: $(TARGET)
 
 # These will be provided to the target
-BUILD := `git rev-parse HEAD`
+BUILD := "`cat VERSION`-`date +%Y%m%d-%H%M%S`+`git rev-parse --short HEAD`"
```
Comment (arnisoph) [inline Makefile]: not trimming trailing newlines could be dangerous. could you fix this please? :)

Comment (invadersmustdie) [reply Makefile]: tr -d '\n' FTW. should be good enough for the beginning.

**Pull request merged**